### DISCUSSION TITLE
Linux Desktop Port?

### DISCUSSION THREAD

**Discussion by ThisNekoGuy** [btimofeev/UniPatcher#37] [Ideas]
Wanted to ask if this could be ported as an app for Linux desktop, since it has some useful features I haven't seen elsewhere...

**Comment by btimofeev** [comment]
What features are you missing on your desktop?  Patchers for all these formats can be found on the Internet.

**Comment by ThisNekoGuy** [comment]
I have yet to see a single unified or independent patcher at all for Linux.
I know Windows has a few, but they all scattered (non-unified) as well and, of course, aren't for any other OS.

Not even going to say anything about Macs because it's far cheaper to use literally anything else, and to my understanding, supposedly harder to develop for than other platforms...

Regardless, that leaves users such as myself in a weird spot because we have all of the emulators but nothing to patch our stuff with.
(And in some cases, nothing [reliable] to rip our stuff with either: namely GC/Wii, PS2[?], and PS3, but that's a bit off-topic.)

**Comment by btimofeev** [comment]
It's real, but I'm not sure if it's worth it.  The code is written in Java and you will have to install the JVM on Linux. 

I once made a port of UniPatcher to a Linux console application for myself.  But I haven't published it and it doesn't match the current sources for a long time.

**Comment by ThisNekoGuy** [comment]
I would try to port the code using a language I know (I'm a junior dev) but I have no experience with this stuff and shuffling through the source didn't reveal a whole lot to me (I don't know much Java)...
